include/deal.II/matrix_free/mapping_info.templates.h

```diff
@@ -1152,10 +1152,12 @@ namespace internal
                 typename VectorizedDouble>
       void
```

Comment:
When looking at the code, I realized that the documentation for the other function below (dealing with faces), see comment https://github.com/dealii/dealii/pull/16317/files#r1413627125, seems to be the same as here for the cells? This could be something one wants to change in the future. Of course this is somewhat unrelated to the present PR.